ondine/hw6-tarot.js

```diff
@@ -0,0 +1,193 @@
+// This IIFE will define a superclass:
```

Comment:
The sections you had time to get to look good! Feel free to drop a line with any questions about other sections.